backends/ebpf/psa/externs/ebpfPsaRegister.cpp

```diff
@@ -180,11 +180,28 @@ void EBPFRegisterPSA::emitInstance(CodeBuilder *builder) {
                                    valueTypeName, size);
 }
 
+cstring EBPFRegisterPSA::getParamName(CodeBuilder* builder, const IR::Expression *expr,
```

Comment:
I would document this function, the name is somewhat misleading.
It really makes up a variable that holds the value of the index or value used to access a register.

Reply:
So has there been a pass before this one which makes sure that any complex expression used to index in a register has been substituted by a temporary?